recipe/deploy/cleanup.php

```diff
@@ -18,7 +18,7 @@
     }
 
     foreach ($releases as $release) {
-        run("ionice -c 3 nice -19 find {{deploy_path}}/releases/$release/ -delete", $runOpts);
+        run("find {{deploy_path}}/releases/$release/ -delete", $runOpts);
```

Comment:
So what is performance gain? rm rf vs find?

Reply:
I found the problem with my deploy, it was mount option **discard**. So I will close request.